operations/helm/charts/mimir-distributed/values.yaml

```diff
@@ -3047,6 +3047,77 @@ minio:
   # Changed the mc config path to '/tmp' from '/etc' as '/etc' is only writable by root and OpenShift will not permit this.
   configPathmc: "/tmp/minio/mc/"
 
+kafka:
+  # -- Enable Kafka for ingest-storage architecture (demo purposes only)
+  enabled: false
+
+  image:
+    # -- The container image registry
+    registry: docker.io
+    # -- The container image repository
+    repository: apache/kafka-native
+    # -- The container image tag
+    tag: 4.0.0
+    # -- The container image pull policy
+    pullPolicy: IfNotPresent
+
+  # -- Kafka service port
+  service:
+    port: 9092
+    controllerPort: 9093
+    # -- Service type
+    type: ClusterIP
+    # -- Service annotations
+    annotations: {}
+    # -- Service labels
+    labels: {}
+  # -- Kafka persistence configuration
+  persistence:
+    enabled: true
+    size: 5Gi
```

Comment:
is the kafka running without any retention settings? can we set the retention to like 24h? I'm also not sure if 5GB won't fill up quickly for almost all use cases. can we use something like 10GB - 5x the ingester disk size (this is a gut feeling, sooo if you disagree, we can stick with 5GB)

Reply:
(i'm also inclined to make the retention configurable)